app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -91,6 +91,10 @@ class BrowserRobot {
         verifyUrl("play.google.com/store/apps/details?id=org.mozilla.fenix")
     }
 
+    fun verifyPrivacyNoticeURL() {
```

Comment:
Why not use verifyUrl("mozilla.org/en-US/privacy/firefox") directly in the test, instead?

Reply:
Done !